adapt.json

```diff
@@ -1,27 +1,27 @@
 {
     "dependencies": {
-        "adapt-contrib-vanilla": ">=3",
-        "adapt-contrib-text": ">=2.0.3",
-        "adapt-contrib-narrative": ">=2.1",
-        "adapt-contrib-media": ">=3",
-        "adapt-contrib-hotgraphic": ">=2.1.1",
-        "adapt-contrib-blank": ">=2.0.1",
-        "adapt-contrib-accordion": ">=2.1.1",
-        "adapt-contrib-graphic": ">=2.0.5",
-        "adapt-contrib-matching": ">=2.1.4",
-        "adapt-contrib-textInput": ">=2.1.1",
-        "adapt-contrib-mcq": ">=2.1.2",
-        "adapt-contrib-gmcq": ">=3.0.0",
-        "adapt-contrib-assessment": ">=2.1.2",
-        "adapt-contrib-assessmentResults": ">=2.1.1",
-        "adapt-contrib-trickle": ">=2.1.5",
-        "adapt-contrib-tutor": ">=2.1.2",
-        "adapt-contrib-boxMenu": ">=2.0.7",
-        "adapt-contrib-pageLevelProgress": ">=2.0.9",
-        "adapt-contrib-resources": ">=2.1.1",
-        "adapt-contrib-slider": ">=2.2.0",
-        "adapt-contrib-spoor": ">=3",
-        "adapt-contrib-bookmarking": ">=2.1.2",
-        "adapt-contrib-languagePicker": ">=2"
+        "adapt-contrib-vanilla": ">=5.0.0",
```

Comment:
These should really be `>=5` etc.

Semver _ranges_ don't require all digits; they are implied. Only the version string in the bower JSON needs all three parts ([reference](https://www.npmjs.com/package/semver#x-ranges-12x-1x-12-)).